packages/rs-drive-abci/src/execution/engine/run_block_proposal/mod.rs

```diff
@@ -63,10 +81,10 @@ where
             0 => self.run_block_proposal_v0(
                 block_proposal,
                 known_from_us,
-                epoch_info.into(),
+                epoch_info,
```

Comment:
We shouldn't pass EpochInfo from previous version to new version of run block proposal